### PR TITLE
[UI] Center text vertically

### PR DESCRIPTION
**Pull request by naimiii**
Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)
@helfi92  @djmitche 

Closes https://github.com/taskcluster/taskcluster/issues/1588.

### CODE REVIEW

File: ui/src/components/TaskRunsCard/index.jsx [hunk 1]
```diff
@@ -94,6 +94,11 @@ const DOTS_VARIANT_LIMIT = 5;
       display: 'inline-flex',
       flexBasis: '50%',
     },
+    articleName: {
```
Comment (helfi92) [inline ui/src/components/TaskRunsCard/index.jsx]: Did you mean `artifactName`? :)

**Comment by naimiii** [comment]
Yeah. Sorry for the mis spelling. Let me fix it now .

File: ui/src/components/TaskRunsCard/index.jsx (hunk 1 above)
Comment (naimiii) [reply ui/src/components/TaskRunsCard/index.jsx]: @helfi92  Yeah .sorry for the mis-spelling. Let me fix it now :) . Really sorry

Comment (naimiii) [reply ui/src/components/TaskRunsCard/index.jsx]: @helfi92  I fixed it .

**Comment by naimiii** [comment]
@helfi92  You are welcome . :)

**Pull request merged**